README.md

```diff
@@ -98,6 +98,40 @@ export default App;
 
 Now, when you will launch your application and shake the device, it will display automatically Netwatch.
 
+If you want activate Netwatch with a button from your app, you **must** disable shake and instead pass the props onPressClose and visible.
```

Comment:
We can create a section "How to display Netwatch" and explain the ways to do that

Reply:
Why we cannot use both? 🤔

Reply:
The shake sets internal state of the component where button passes props.
I found a bug when I was using button and shake simultaneously. The button become inactive or need to be clicked two times to resynchronize with internal state. That's why at this time, I prefer avoid the case where button and shake are active at the same time.
I'll try to improve this feature however I think it's not the priority.

For the section, I can create a new one